src/methods.jl

```diff
@@ -13,6 +13,9 @@ function fits_indices(string::String)
     return reverse(idxs)
 end
 
+# Converts Int to Float and leaves Float to Float itself
+convert_value(x, S) = S <: Integer ? round(S, x) : convert(S, x)
```

Comment:
Why not defining two methods
```suggestion
convert_value(x, S::Integer) = round(S, x)
convert_value(x, S) = convert(S, x)
```
?

Note also that traditionally, the type argument is the first one, as you can see in the `round` and `convert` methods you're calling here

Reply:
Yes, much better than my suggestion!

Reply:
Switched to conventional argument style, and the functions look something like:
```
convert_value(S::Integer, x) = round(S, x)
convert_value(S, x) = convert(S, x)
```
The function call `convert_value(Int16, 5.4)` goes to the second function, whereas it should have gone to the first one, why is it happening? :thinking:

Reply:
haha, we both had a folly-
```julia
convert_value(S::Type{<:Integer}, x)
```